extras/docker/Dockerfile

```diff
@@ -1,28 +1,61 @@
 FROM ubuntu:16.04
 
-RUN apt-get update && apt-get dist-upgrade -y && apt-get autoremove -y && apt-get clean && apt-get install -y \
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install YETI dependencies
+RUN apt-get update && apt-get upgrade -y && apt-get autoremove -y && apt-get install -y --no-install-suggests --no-install-recommends \
   build-essential \
   python-dev \
   libxml2-dev \
   libxslt-dev \
   zlib1g-dev \
-  python-virtualenv \
   python-pip \
+  python-setuptools \
+  python-dev \
+  python-wheel \
+  locales \
   git \
-  locales
+  libmagic1 \
+  curl \
+  apt-transport-https \
+  uwsgi-plugin-python \
+  uwsgi && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/* /usr/share/man/* /root/.cache/*
 
-RUN locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LC_ALL en_US.UTF-8
+# Setup locales
+RUN locale-gen en_US.UTF-8 && \
+        echo "LC_ALL=en_US.UTF-8" >> /etc/environment && \
+        echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen && \
+        echo "LANG=en_US.UTF-8" > /etc/locale.conf
 
+# Install frontend dependencies
+RUN apt-get update && apt-get install -y --no-install-suggests --no-install-recommends gnupg2 && \
```

Comment:
I don't know much about docker but why not merge this with the command on L6?

Reply:
I left it this way just to keep a logical structure within the Dockerfile for maintenance reasons.By merging this line with L6 we would lose this layer at the final image and save a few MB. I don't mind merging it at all, i can update the PR tomorrow.